src/api/helpers.js

```diff
@@ -0,0 +1,5 @@
+const BASE_URL = 'http://www.omdbapi.com/?apikey=41fc156d&t=';
```

Comment:
![image](https://user-images.githubusercontent.com/62404765/112851047-42d91200-90b3-11eb-8da0-f3ba8f2adb94.png)
```suggestion
const BASE_URL = 'https://www.omdbapi.com/?apikey=41fc156d&t=';
```